deepCR/evaluate.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from deepCR.util import maskMetric
+from deepCR.dataset import dataset
+from tqdm import tqdm as tqdm
+
+__all__ = 'roc'
+
+
+def _roc(model, data, thresholds):
+    """ internal function called by roc
+
+    :param model:
+    :param data: deepCR.dataset object
+    :param thresholds:
+    :return: tpr, fpr
```

Comment:
note that these parameters are returned in percentages not decimals (since you multiply them by 100).